### PR TITLE
[helper] Switch from print to log

### PR DESCRIPTION
**Pull request by jonathanmetzman**
This is better because:
1. It avoids the need to print 'ERROR:' before some messages.
2. We don't need to specify writing to stderr, this is the default.

### CODE REVIEW

File: infra/helper.py [hunk 1]
```diff
@@ -72,6 +73,8 @@ def main():  # pylint: disable=too-many-branches,too-many-return-statements
   if not os.path.exists(BUILD_DIR):
     os.mkdir(BUILD_DIR)
 
+  logging.basicConfig(level=logging.DEBUG)
```
Comment (inferno-chromium) [inline infra/helper.py:76]: default should be INFO ?

**Review by inferno-chromium — APPROVED** [review]

File: infra/helper.py [hunk 2]
```diff
@@ -756,10 +757,8 @@ def download_corpora(args):
     with open(os.devnull, 'w') as stdout:
       subprocess.check_call(['gsutil', '--version'], stdout=stdout)
   except OSError:
-    print(
-        'ERROR: gsutil not found. Please install it from '
-        'https://cloud.google.com/storage/docs/gsutil_install',
-        file=sys.stderr)
+    logging.error('ERROR: gsutil not found. Please install it from '
```
Comment (oliverchang) [inline infra/helper.py:760]: no need for "ERROR" in the message

**Review by oliverchang — APPROVED** [review]

File: infra/helper.py (hunk 1 above)
Comment (jonathanmetzman) [reply infra/helper.py]: Done.

File: infra/helper.py (hunk 2 above)
Comment (jonathanmetzman) [reply infra/helper.py]: Done.

**Pull request merged**